14-wt-machine-learning.Rmd

```diff
@@ -71,6 +85,8 @@ The outcome was the final course grade that the student earned. The predictor va
 
 To interpret our findings, we will  three main factors: (1) predictive accuracy of the random forest model, (2) variable importance, and (3) variance explained by the final random forest model. In this walkthrough, we will the R package **caret** to carry out the analyses.
 
+## Load Packages
```

Comment:
Was this intentional? ##Load packages and then ##Analysis? It looks like they are back to back

Reply:
Addressed!